### DISCUSSION TITLE
Projects with low Build Activity - Survey

### DISCUSSION THREAD

**Discussion by jadsonjs** [swc-project/swc#3112] [General]
Dear Developers of project **swc-project/swc**,

We are conducting a study on CI metrics and we noticed that in your project, something peculiar happened.

During the period you are using Travis CI, the project passed by a long period of inactivity.
We noticed that the TravisCI configuration file (.travis.yml) was committed to your project on the date of **2017-12-22**.
But there is only a build history in TravisCI for a very short period, between **2019-12-11** and **2020-01-03**, in which were performed suddenly **128** builds.

Please, we would like you to answer this short survey with just 4 questions to help us identify the cause of this behavior.

~link removed~

**Comment by kwonoj** [comment]
Peeking history looks like this same message is spreaded over various repository and there is no good way to ensure trust for the given link. I have removed links from post and locked this discussion.
